<launch>

    <!-- joint_state_publisher related -->
    <arg name="jsp_gui" default="false" />
	<rosparam param="source_list">["/servo_node/joint_states", "/servo/joint_states"]</rosparam>
    <node if="$(arg jsp_gui)" name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
    <node unless="$(arg jsp_gui)" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />

    <param name="robot_description" command="$(find xacro)/xacro '$(find scout_description)/urdf/mini.xacro' urdf_extras:='$(find yahboom)/urdf/attachments.urdf.xacro'" />

    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />

    <node name="rviz" pkg="rviz" type="rviz" args="-d '$(find yahboom)/rviz/display_robot.rviz'" required="true" />

</launch>
